GVRf/Framework/framework/build.gradle

```diff
@@ -108,6 +108,10 @@ android {
     sourceSets.main {
         java.srcDirs = ['src/main/java']
         resources.srcDir 'src/main/libs'
+
+        jniLibs {
```

Comment:
I don't see any of these binaries in the final apk. Are they really needed?

Reply:
Thanks